actors/builtin/reward/reward_actor.go

```diff
@@ -93,12 +93,15 @@ func (a Actor) AwardBlockReward(rt vmr.Runtime, params *AwardBlockRewardParams)
 	AssertMsg(big.Add(rewardPayable, penalty).LessThanEqual(priorBalance),
 		"reward payable %v + penalty %v exceeds balance %v", rewardPayable, penalty, priorBalance)
 
+	// if this fails, we can assume the miner is responsible and do not to fail here
```

Comment:
"If this fails, we can assume the miner is responsible and avoid failing here."

Reply:
This is done.

Reply:
This is done.